core/tests/protractor/accessibility.js

```diff
@@ -54,3 +54,40 @@ describe('Cache Slugs', function() {
     general.checkConsoleErrorsExist(expectedErrors);
   });
 });
+
+describe('Meta Tags', function() {
+  var GET_STARTED_URL = '/get_started';
+  var GET_STARTED_META = {
+    name: 'Personalized Online Learning from Oppia',
+    description: 'Learn how to get started using Oppia.'
+  };
+  beforeEach(function() {
+    browser.get(GET_STARTED_URL);
+    waitFor.pageToFullyLoad();
+  });
+
+  it('should set the correct itemprop meta tags', function() {
+    var nameMeta = element(by.css('meta[itemprop="name"]'));
```

Comment:
Any direct references using element(by.css) etc. should go into protractor_utils/, into a pageobject (GetStartedPage.js, say). You can expose a .getMetaTagContent() function and use that here. See other e2e tests for details.

Reply:
Done